applications/machine_learning/src/modules/usb_state.c

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <usb/usb_device.h>
+
+#define MODULE usb
+#include <caf/events/module_state_event.h>
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(MODULE, CONFIG_ML_APP_USB_LOG_LEVEL);
+
+
+static bool handle_module_state_event(const struct module_state_event *event)
+{
+	if (check_state(event, MODULE_ID(main), MODULE_STATE_READY)) {
+
+		if (!usb_enable(NULL)) {
```

Comment:
I would suggest
```
if (check_state....) {
   int err = usb_enable(NULL);

   if (!err) {
      .....
   }
}
```

the way it is now suggests that `usb_enable` returns boolean and you start to thing why we want it to be false...

alternatively compare result to 0 i.e.
```
   if (usb_enable(NULL) == 0) {
      ...
   }
```

Reply:
Fixed as suggested.